src/controllers/info.ts

```diff
@@ -1,20 +1,34 @@
 import { ParameterizedContext } from 'koa';
 
 import TMDBConfiguration from '../models/TMDBConfiguration';
-import { moviedb } from '../services/tmdb-api';
+import { tmdb } from '../services/tmdb-api';
+
+let configuration: { imageBaseURL: string };
+let configurationExpiryDate: Date;
 
 export const getTMDBImageBaseURL = async(): Promise<string> => {
-  const configuration = await TMDBConfiguration.findOne().lean()
-    .exec();
-  if (configuration) {
+  // 1) Return in-memory value unless it has expired
+  const today = new Date();
+  if (configuration && configurationExpiryDate && today < configurationExpiryDate) {
```

Comment:
We could put this in the model to avoid bloat here. Guess it doesnt matter too much